plugins/discovery/pod_test.go

```diff
@@ -0,0 +1,56 @@
+package discovery
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/wavefronthq/wavefront-collector-for-kubernetes/internal/discovery"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+	"testing"
+)
+
+type DummyDiscoverer struct{}
```

Comment:
Didn't think it was worth moving this to the utils folder if its only used here. Would love a second opinion on that decision

Reply:
I agree.